### PR TITLE
Modifying README.md to address ISSUE-53

### PR DESCRIPTION
**Pull request by bmunroezappos**

### CODE REVIEW

**Comment by ruslanrusu** [comment]
here is what i needed to add

`GRANT SELECT ON workflow to retool_readonly;
GRANT SELECT ON workflow_run to retool_readonly;
GRANT SELECT ON workflow_save to retool_readonly;`

File: usage-viewer/README.md [hunk 1]
```diff
@@ -48,6 +49,7 @@ GRANT SELECT ON resources to <YOUR_READ_ONLY_USER>;
 GRANT SELECT ON groups to <YOUR_READ_ONLY_USER>;
 GRANT SELECT ON resources to <YOUR_READ_ONLY_USER>;
 GRANT SELECT ON organizations to <YOUR_READ_ONLY_USER>;
+GRANT SELECT ON workflow_save to <YOUR_READ_ONLY_USER>;
```
Comment (mattharris-stripe) [inline usage-viewer/README.md:52]: Looks like the app also needs these extra tables, too:

```suggestion
GRANT SELECT ON workflow_save to <YOUR_READ_ONLY_USER>;
GRANT SELECT ON workflow to <YOUR_READ_ONLY_USER>;
GRANT SELECT ON workflow_run to <YOUR_READ_ONLY_USER>;
```

**Review by mattharris-stripe — COMMENTED** [review]
Need two more table grants, but otherwise, it would be great to merge this in!

**Comment by julieechenn** [comment]
Good catch Ruslan and Matt! Accepted & merged

**Comment by bmunroezappos** [comment]
Thanks everyone!

**Pull request merged**